lib/new_relic/language_support.rb

```diff
@@ -78,6 +78,11 @@ def camelize(string)
       camelized.split(/\-|\_/).map(&:capitalize).join
     end
 
+    def camelize_with_first_letter_downcased(string)
+      camelized = camelize(string)
+      camelized[0].downcase.concat(camelized[1..-1])
```

Comment:
Coming in Rails v7.1...

https://edgeapi.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-downcase_first